public/app/features/explore/ExploreToolbar.tsx

```diff
@@ -91,7 +92,8 @@ export function ExploreToolbar({ exploreId, onChangeTime, onContentOutlineToogle
   const correlationDetails = useSelector(selectCorrelationDetails);
   const isCorrelationsEditorMode = correlationDetails?.editorMode || false;
   const isLeftPane = useSelector(isLeftPaneSelector(exploreId));
-  const { drawerOpened, setDrawerOpened, queryLibraryAvailable } = useQueriesDrawerContext();
+  const { drawerOpened, setDrawerOpened } = useQueriesDrawerContext();
+  const { queryLibraryAvailable } = useQueryLibraryContext();
```

Comment:
i know it was like this already, but we're a little inconsistent with this. in some places we use the context value, in some places we directly reference the feature toggle 🤔 my slight preference would be to remove this from the context and just use the feature toggle value directly where needed